src/Xamarin.Android.Build.Utilities/Sdks/AndroidSdkWindows.cs

```diff
@@ -24,27 +24,36 @@ class AndroidSdkWindows : AndroidSdkBase
 		public override string NdkHostPlatform64Bit { get { return "windows-x86_64"; } }
 		public override string Javac { get; protected set; } = "javac.exe";
 
+		string GetMDRegistryKey ()
+		{
+			var regKey = Environment.GetEnvironmentVariable ("XAMARIN_ANDROID_REGKEY");
+			return string.IsNullOrWhiteSpace (regKey) ? MDREG_KEY : regKey;
+		}
+
 		public override string PreferedAndroidSdkPath {
 			get {
 				var wow = RegistryEx.Wow64.Key32;
-				if (CheckRegistryKeyForExecutable (RegistryEx.CurrentUser, MDREG_KEY, MDREG_ANDROID_SDK, wow, "platform-tools", Adb))
-					return RegistryEx.GetValueString (RegistryEx.CurrentUser, MDREG_KEY, MDREG_ANDROID_SDK, wow);
+				var regKey = GetMDRegistryKey ();
+				if (CheckRegistryKeyForExecutable (RegistryEx.CurrentUser, regKey, MDREG_ANDROID_SDK, wow, "platform-tools", Adb))
```

Comment:
Should this check for *both*?

Suppose (1) `%XAMARIN_ANDROID_REGKEY%` is set to a non-empty string, and (2) that registry location *does not exist*. What should be done?

I think it would make sense to try `%XAMARIN_ANDROID_REGKEY%`, and if the registry location doesn't exist, *also* try `MDREG_KEY`.

We should also log which locations we're trying, if that's not already done.